.github/workflows/patch_verification.yml

```diff
@@ -9,7 +9,7 @@
 # github.com also has a powerful web editor that can be used without
 # committing.
 
-name: checkpatch
+name: patch_verification
```

Comment:
How about `codestyle.yml`? Unlike `checkpatch`, `yamllint` does not look at the patch.